README.md

```diff
@@ -1,6 +1,6 @@
 # KAssert: Karlsruhe Assertion Library for C++
 <p align="center">
-   <img src="https://github.com/kamping-site/kassert/blob/main/docs/kassert-logo.svg"\>
+   <img src="https://raw.githubusercontent.com/kamping-site/kassert/docs/images/logo.svg"\>
```

Comment:
This diff confuses me. But it seems like the result is correct? Right now, the image doesn't show. But that might be because it's not pushed to main yet.

Reply:
This is where GitHub serves the raw version of files from. The other ist the web interface address. Raw should have less overhead.